packages/workflow/src/records/state-transitions.ts

```diff
@@ -415,7 +415,7 @@ export async function toRegistered(
   )
 
   /* Setting registration number here */
-  const system = `${OPENCRVS_SPECIFICATION_URL}id/${
+  const system = `http://opencrvs.org/specs/id/${
```

Comment:
is there a reason why we'd rather do this instead of the OPENCRVS_SPECIFICATION_URL constant?

Reply:
Not really, just thought it would be easier to read, also in our recent state transition tasks' extension url, we haven't quite used the `OPENCRVS_SPECIFICATION_URL` constant. It was only being used here. So, that's why.

Reply:
Okay 👍🏻 It's fine, if we wanna align these URLs we can do it all together in the parent feature PR.